src/main/java/by/itechart/libmngmt/controller/command/commands/AddBookPageCommand.java

```diff
@@ -0,0 +1,37 @@
+package by.itechart.libmngmt.controller.command.commands;
+
+import by.itechart.libmngmt.controller.command.LibraryCommand;
+import by.itechart.libmngmt.dto.BookDto;
+import by.itechart.libmngmt.dto.BookPageDto;
+
+import javax.servlet.ServletException;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.Arrays;
+
+public class AddBookPageCommand extends LibraryCommand {
+    private static AddBookPageCommand instance;
+
+    public static AddBookPageCommand getInstance() {
+        if(instance == null){
```

Comment:
this singletone implementation is not threadsafe. Read https://habr.com/ru/post/129494/ and improve